privval/priv_validator_test.go

```diff
@@ -25,10 +25,13 @@ func TestGenLoadValidator(t *testing.T) {
 	height := int64(100)
 	privVal.LastHeight = height
 	privVal.Save()
-	addr := privVal.GetAddress()
+	addr, err := privVal.GetAddress() // FilePV doesn't err here
```

Comment:
don't think comment is necessary if we're handling the error

also, it's not quite true. FilePV can error (in 0.1% cases due to file corruption or some other rare event)

Reply:
BTW, while it's true that due to file corruption FilePv might error, it won't error on that specific function call (but it will fail earlier and exit in `LoadFilePV`). Once a FilePV object is loaded it won't err on `GetAddress`.